fenix/app/nimbus.fml.yaml

```diff
@@ -326,6 +326,14 @@ features:
         type: List<OnboardingPanel>
         default: ["themes", "toolbar-placement", "sync", "tcp", "privacy-notice"]
 
+  glean:
+    description: "A feature that provides server-side configurations for Glean metrics (aka Server Knobs)."
+    variables:
+      glean-metric-configuration:
```

Comment:
Nit: `glean-` is redundant, and `configuration` is somewhat over-general.

Suggest `metrics-enabled`.